### DISCUSSION TITLE
[Mini-Event] Statistiques Personnels

### DISCUSSION THREAD

**Discussion by MF4db** [Crownicles/Crownicles#909] [Nouvelles idées]
Comme son nom l'indique ce serait des sortes de minis-events te donnant tes statistiques comme : le nombre de potions bus, le nombre de morts, le nombre total de trajets effectués, l'argent total gagné et dépensé ect.

**Comment by ghost** [comment]
ha ouais ce serait grave cool 😃

**Comment by BastLast** [comment]
On y a déjà beaucoup pensé mais à l'heure actuelle on ne collecte pas ces statistiques du coup c'est impossible à calculer, les implémenter (correctement) prendra du temps, cette suggestion sera implémenté le jour ou ou l'aura fait ( et on le fera forcément car ca peut être utile pr d'autres choses tel que l'anti cheat)
